test/extended/scheduling/pods.go

```diff
@@ -112,7 +112,9 @@ var _ = g.Describe("[sig-scheduling][Early]", func() {
 	oc := exutil.NewCLI("scheduling-pod-check")
 
 	g.BeforeEach(func() {
```

Comment:
@ingvagabund do you recall why this test does not have `apigroup:apps.openshift.io` apigroup? And why the `The openshift-apiserver pods` test has bunch of apigroups as well as this one?